### ISSUE TITLE
Crash (java fatal)

### ISSUE THREAD

**Issue by soshimee** [Axeryok/CocoaInput#13]
Java crashes with a fatal error when I try to type anything in chat in Korean.

Input method: IBus
OS: Arch Linux
Minecraft version: 1.17

This is all the information I can provide since it is a java fatal error and it contains some critical information.

**Comment by Axeryok** [comment]
I'm not sure of best solution because of lack of logs but I think that AdvancedPreeditDraw caused crash. 
Read spoiler on distribution page and turn AdvancedPreeditDraw to false.

**Comment by soshimee** [comment]
Yes, AdvancedPreeditDraw was the culprit, thanks for the information. What does it exactly do though?

**Comment by Axeryok** [comment]
APD shows cursor location and which phrase is converting visually and it is important for Japanese IME.
But my implementation is not so compatible for Korean IME(on Linux especially).
In addition, I guess that APD is not important at all for  Korean IME.

**Comment by soshimee** [comment]
I believe, since I don't see any issues with it, except that cursor shows like |_ instead of ᅟᆜ which the latter is normal.
